magsnake.md

```diff
@@ -0,0 +1,134 @@
+  1pcs 4 digit 7-segment Display
+  1pcs Sound Sensor Module
+  1pcs LCD1602 Blue Backlight with Soldering
+  1pcs IC L293D
+  1pcs IC 74HC595
+  1pcs Active Buzzer
+  1pcs Passive Buzzer
+  1pcs RTC Module
+  1pcs DHT11 Temperature and Humidity Module
+  2pcs Potentiometer
+  1pcs Rotary Encoder Module
+  1pcs Joystick Module
+  1pcs Keypad Module
+  1pcs 5V Relay
+  1pcs IR Receiver Module
+  1pcs MEGA2560 Controller Board
+  1pcs Breadboard
+  1pcs Servo Motor (SG90)
+  1pcs Stepper Motor
+  1pcs ULN2003 Stepper Motor Driver Board
+  1pcs Prototype Expansion
+  1pcs Power Supply Module WARNING: Pls. do not use the voltage higher than 9V
+  1pcs HC-SR501 PIR Motion Sensor
+  1pcs Ultrasonic Sensor
+  1pcs GY-521 Module ( with pin header)
+  1pcs 3V Servo Motor
+  1pcs MAX7219 Module
+  1pcs Remote
+  1pcs 9V 1A Power Supply
+  1pcs 65 Jumper Wire
+  1pcs Water Lever Sensor
+  1pcs USB Cable
+  1pcs 9V Battery connector with DC jack
+  1pcs RC522 RFID Module
+  120pcs Resistor (10R/100R/220R/330R/1K/2K/5K1/10K/100K/1M)
+  20pcs Female-to-male Dupont Wire
+  
+  **1 x Orange Raspberry Pi Advance Kit Includes:**
+
+  1 x 5V 3A Power supply with ON/OFF Switch
+  1 x 3.5-inch Touchscreen LCD Display with stylus
+  1 x MB-102 Breadboard
+  1 x Bread Board Power Supply Module
+  1 x SanDisk 32GB class 10 SD Card
+  1 x Raspberry Pi Camera 
+  1 x SD Card Adapter
+  1 x USB to TTL Converter Cable
+  1 x Black Plastic Raspberry PI case
+  1 x Compatible to case Cooling FAN
+  1 x Micro HDMI Male To HDMI Female  Adapter
+  1 x LAN Cable (3-meter length)
+  1 x HDMI TO HDMI Cable (1-meter Length)
+  1 x 8 channel 3.3v to 5V Level Converter
+  1 x 40 pins Female to Female Dupont Cable 20 cm
+  1 x 40 pins Male to male Dupont Cable – 20 cm
+  1 x 40 pins Male to Female Dupont Cable – 20 cm
+  1 x Red LED – 5pcs.
+  1 x Yellow LED – 5pcs.
+  1 x Blue LED – 5pcs.
+  1 x 5mm Push Buttons – 5pcs
+  1 x 220/320/1k/2k resistors- 5pcs (each)
+  1 x Heart Rate Sensor Module
+  1 x Active Buzzer Module
+  1 x IR Sensor Module
+  1 x HC-SR04 Ultrasonic Distance Sensor Module
+  1 x Capacitive Touch Sensor Module
+  1 x ADS1115 16Bit I2C- 4 Channel ADC Module
+  1 x 5V DC Stepper Motor
+  1 x Stepper Motor Driver
+  1 x RFID Card and Reader with 1 RFID Tag
+  
+
+## Budget
+
+                                                                                                   (The Conversion of currencies is according to 1 Jan, 12:29 am UTC)
+
+| Product                     | Supplier/Link                                                           | Cost                    |
+| --------------------------- | ----------------------------------------------------------------------- | ----------------------- |
+| Arduino Mega 2560           | https://robu.in/product/original-arduino-mega-2560-atmega2560-mcu-rev3/ | ₹ 2,885.00 ($34.86)     |
+| Raspberry Pi 4 Model-B      | https://robu.in/product/raspberry-pi-4-model-b-with-8-gb-ram/           | ₹ 7,999.00 ($96.66)     |
```

Comment:
it looks like this is out of stock? is there another place where you'd be able to secure a raspberry pi?

Reply:
It is available for those who have joined waitlist for 1 week.
I have joined the waitlist already and it is available in 1-2 days.
Robu.in is the official supplier of Pi.

Reply:
ahhh coool, awesome!!